src/components/FeaturedPosts.js

```diff
@@ -0,0 +1,157 @@
+import Slider from 'react-slick';
+import {css} from '@emotion/core';
+import {map} from 'ramda';
+import {mapNodeToProps} from '~/utilities';
+import {LOCHMARA_COLOR, mq} from '~/constants';
+import Img from 'gatsby-image';
+import {ExternalLink, Text} from '~/components';
+import {Link} from 'gatsby';
+
+const settings = {
```

Comment:
Can this be abstracted out, e.g. would be if we use similar settings somewhere else, or a global constants file we can place this?

Reply:
We wouldn't be using this elsewhere. Should probably keep it centralized.